### DISCUSSION TITLE
For some reason uuid library started failing

### DISCUSSION THREAD

**Discussion by lewysigns** [anvilistas/anvil-extras#372] [General]
I am receiving this error on import of the uuid library

```ExternalError: TypeError: Failed to fetch dynamically imported module: https://jspm.dev/uuid@8.3.2```

Not sure why this started occurring as the import line has not changed.

Any ideas?

**Comment by rhurlbatt** [comment]
I am not seeing this error in any of the apps that use this module.
Has any code changed recently? Have you tried to create a minimal standalone app to replicate the behaviour - this is a good way to verify its not some other code impacting the main app.

**Comment by s-cork** [comment]
@lewysigns - no problems for me either

It could be that jspm.dev was down when you were trying it.
Happy to include a fallback cdn if the first one fails.

To test if the url is generally working just visit the url in a new tab to see what happens
You'll then get a link to another url - in my case `/npm:uuid@8.3.2!cjs` and you can visit the full url `https://jspm.dev/npm:uuid@8.3.2!cjs` to see the javascript code.

Are you still having this problem?

**Reply by lewysigns** [reply]
sorry for the dead response. I'll review today if this problem is still existing. I resorted to using a different library at the moment.

Thanks.
